permits/forms.py

```diff
@@ -1424,3 +1466,8 @@ def save(self, request):
             raise ProviderException(_("Unknown social account provider"))
 
         return adapter.save_user(request, self.sociallogin, form=self)
+
+
+class AnonymousRequestForm(forms.Form):
+    required_css_class = "required"
+    captcha = CaptchaField(required=True)
```

Comment:
Maybe use a different `label` or `help_text` to explain what Captcha is to the user

Reply:
Je ne sais pas exactement quoi écrire et il me semble que "Captcha" est entré dans le langage courant...

Reply:
Ok pour moi, je laisse le jugement à la sensibilité du customer 😄